cypress/e2e/PatientCreation/index.spec.ts

```diff
@@ -0,0 +1,108 @@
+import moment from 'moment';
+
+function buildRAMQ(ramqSuffix: number) {
+  const ramqPrefix = 'TESC';
+  return `${ramqPrefix}${moment().format('YYMMDD')}${ramqSuffix.toLocaleString('fr-CA', { minimumIntegerDigits: 2 })}`;
+}
+
+function buildMRN(mrnSuffix: number) {
+  const mrnPrefix = 'CYPRESS';
+  return `${mrnPrefix}${moment().format('YYMMDD')}${mrnSuffix.toLocaleString('fr-CA', { minimumIntegerDigits: 2 })}`;
+}
+
+describe('e2e: Patient Creation', () => {
+  describe('should create a patient', () => {
+    // @ts-ignore
```

Comment:
The `@ts-ignore` here are required 'cause the second param isn't in the types yet..